test/utils.jl

```diff
@@ -10,6 +10,18 @@ task.
 function get_instance_filepaths(problem_name::AbstractString, task::AbstractString)
     model_filepath = joinpath(artifact"uai2014", task, problem_name * ".uai")
     evidence_filepath = joinpath(artifact"uai2014", task, problem_name * ".uai.evid")
+    query_filepath = joinpath(artifact"uai2014", task, problem_name * ".uai.query")
```

Comment:
It would be better to write a function in `src/utils.jl` to help reading the instance from a folder.
e.g. `get_instance_folder(folder, problem_name::String)`, when the input folder is an artifact folder, e.g. `joinpath(artifact"uai2014", task)`, it loads the files in the artifacts folder.

Reply:
@GiggleLiu Now that I think about a it a bit more, I'm not sure whether this is the best idea. The reason is that in many cases the files that correspond to a given "instance", will not be in the same directory. In fact, if you download the UAI problems from the official UAI inference competition website, you will see that the solutions they provide are separated from the other files. It was I who decided that it would be handy to have all these files in the same directory. Other people might want to keep them separated (since the solution is something different than the problem definition).